core/store/Cargo.toml

```diff
@@ -51,11 +51,14 @@ name = "store_bench"
 harness = false
 
 [features]
+cold_storage = []
 default = []
 io_trace = []
 no_cache = []
 single_thread_rocksdb = [] # Deactivate RocksDB IO background threads
-test_features = []
+test_features = [
+  "cold_storage",
```

Comment:
I dunno if we want this. It might be better to have cold_store added to intergation-tests perhaps?

Reply:
I think that’s what flat_state does.  Just run the test manually? Have to ask flat state people I guess.

Reply:
So flat state has a separate buildkite: https://buildkite.com/nearprotocol/nearcore-flat-state

My feeling is that at the moment we don’t want it in CI yet.  And some time later we might add it to nightly.  For now it’s better that CI tests code which is close to production code.